magnum_capi_helm/driver.py

```diff
@@ -62,6 +62,11 @@ def provides(self):
                 "os": "capi-kubeadm-cloudinit",
                 "coe": "kubernetes",
             },
+            {
+                "server_type": "vm",
+                "os": "flatcar",
```

Comment:
What I was assuming here was something like: "capi-kubeadm-ignition"

But yeah, I should catch up with Jake on what he was thinking upstream?